src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/StandardAddressCheckRequest.kt

```diff
@@ -17,6 +20,8 @@ class StandardAddressCheckRequest(
   @ManyToOne
   @JoinColumn(name = "address_id", referencedColumnName = "id")
   val address: Address,
+  @OneToMany(mappedBy = "standardAddressCheckRequest", fetch = FetchType.LAZY, cascade = [CascadeType.ALL], orphanRemoval = true)
+  val residents: MutableSet<Resident> = mutableSetOf(),
```

Comment:
not sure if we need to track insertion order or anything  🤷 , guess this is good for now!

Reply:
good point, me may do for the front end